app/population/view/daily.njk

```diff
@@ -0,0 +1,114 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{%  macro statsEntry(property, value) %}
+  {%  set key = "population::daily_view.labels." + property %}
+  {%  set label = t(key) %}
+
+  <h4 class="govuk-heading-s">{{ label }}</h4>
+  <p class="govuk-body">{{ value }}</p>
+{% endmacro %}
+
+{% macro statsGroup(headingProperty, stats) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  {%  for stat in stats %}
+    {{  statsEntry(stat.property, stat.value) }}
+  {% endfor %}
+{% endmacro %}
+
+{% macro transfersGroup(headingProperty, transfers) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  <p class="govuk-body">
+    {{  transfers }}
+  </p>
+{% endmacro %}
+
+{% set displayDate = details.date if details.date else date | formatDateWithRelativeDay %}
+
+{% block pageTitle %}
+  {{ t("population::daily_view.page_title_with_date", {
+    date: displayDate
+  }) }}
+{% endblock %}
+
+
+{% block content %}
+  <header class="govuk-grid-row govuk-!-margin-bottom-7">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
+        <span class="govuk-caption-xl">
+            {{ t("population::daily_view.subheading") }}
+            </span>
+        <span class="govuk-!-display-block">{{ displayDate }}</span>
+      </h1>
+    </div>
+  </header>
+
+  {%  call appPanel({classes: 'govuk-!-margin-bottom-4'}) %}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+
+        {% if spaces.details.updated_at %}
+          <h2 class="govuk-heading-xl">{{ t("population::daily_view.free_spaces_with_count", {count:spaces.details.free_spaces}) }}</h2>
+          <p class="govuk-body">{{ t("population::daily_view.free_space_description") }}</p>
+          <p class="govuk-body"><b>{{  t("population::daily_view.labels.last_updated") }} {{  spaces.details.updated_at | formatDateWithTimeAndDay }}</b></p>
+          <p class="govuk-body-m">
```

Comment:
I don't think there is a `govuk-body-m` class actually. I think it's the same as [`govuk-body`](https://design-system.service.gov.uk/styles/typography/).

I also had a quick look and the same style is applied to the `p` element so you can actually short all occurrences of `<p class="govuk-body">` or `<p class="govuk-body-m">` to `<p>` in this template.

Reply:
This particular element also has a set style of a bottom margin applied, which is leading to the spacing on the inside of the box to be uneven:

<kbd>![image](https://user-images.githubusercontent.com/3327997/99569386-76e74c80-29c8-11eb-904e-4c41f613c42c.png)</kbd>

It can be resolved by adding `govuk-!-margin-bottom-0` to this element.

Reply:
Removed usage of `govuk-body-m` - I spotted `govuk-body-s` and `govuk-body-l` and figured there was a `govuk-body-m` - I was wrong :)
Removed usaged of govuk-body on `<p>` elements.
Spacing issues resolved on both screens with and without free space data.